phpBB/includes/ucp/ucp_main.php

```diff
@@ -218,6 +238,8 @@ function main($id, $mode)
 					extract($phpbb_dispatcher->trigger_event('core.ucp_main_front_modify_template_vars', compact($vars)));
 
 					$template->assign_block_vars('topicrow', $topicrow);
+
+					$pagination->generate_template_pagination(append_sid("{$phpbb_root_path}viewtopic.$phpEx", "t=$topic_id"), 'topicrow.pagination', 'start', $replies + 1, $config['posts_per_page'], 1, true, true);
```

Comment:
This actually does add pagination for the topics themselves, so that's good:
![image](https://user-images.githubusercontent.com/394418/231263761-efd7a02a-a3f7-4d43-9677-0657f021f385.png)